### ISSUE TITLE
OpenLiberty 21.0.0.12 breaks server startup

### ISSUE THREAD

**Issue by rtclauss** [IBMStockTrader/trade-history#60]
The `Dockerfile` doesn't specify a version of liberty in the `FROM` stanza:
```Dockerfile
...
FROM openliberty/open-liberty:21.0.0.9-kernel-slim-java11-openj9-ubi
...
```

This will pull OpenLiberty 21.0.0.12 which causes the server to not start properly.
```log
[AUDIT ] CWWKE0001I: The server defaultServer has been launched.
[AUDIT ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/keystore.xml
[AUDIT ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/open-default-port.xml
[INFO ] CWWKE0002I: The kernel started after 2.863 seconds
[INFO ] CWWKF0007I: Feature update started.
[ERROR ] CWWKF0001E: A feature definition could not be found for com.ibm.websphere.appserver.javaeePlatform-7.0
[INFO ] Aries Blueprint packages not available. So namespaces will not be registered
[INFO ] Aries Blueprint packages not available. So namespaces will not be registered
[INFO ] CWWKS0007I: The security service is starting...
[AUDIT ] CWWKZ0058I: Monitoring dropins for applications.
[INFO ] DYNA1001I: WebSphere Dynamic Cache instance named baseCache initialized successfully.
[INFO ] DYNA1071I: The cache provider default is being used.
[INFO ] DYNA1056I: Dynamic Cache (object cache) initialized successfully.
[INFO ] CWWKS5775I: The MicroProfile JWT version 1.1 mpConfigProxy processed successfully.
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.app.manager.war [134]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
[INFO ] CWWKS4105I: LTPA configuration is ready after 0.017 seconds.
[INFO ] CWWKS1123I: The collective authentication plugin with class name NullCollectiveAuthenticationPlugin has been activated.
[INFO ] CWPKI0830I: Certificate with the CN=IBM Cloud Databases SubjectDN from the cert_defaultTrustStore environment variable is being added to the defaultTrustStore keystore.
[INFO ] CWPKI0830I: Certificate with the CN=cluster-ca v0,O=io.strimzi SubjectDN from the cert_defaultTrustStore environment variable is being added to the defaultTrustStore keystore.
[INFO ] CWPKI0830I: Certificate with the CN=Amazon RDS Root 2019 CA,OU=Amazon RDS,O=Amazon Web Services\, Inc.,ST=Washington,L=Seattle,C=US SubjectDN from the cert_defaultTrustStore environment variable is being added to the defaultTrustStore keystore.
[INFO ] CWPKI0830I: Certificate with the CN=Amazon RDS us-east-2 2019 CA,OU=Amazon RDS,O=Amazon Web Services\, Inc.,L=Seattle,ST=Washington,C=US SubjectDN from the cert_defaultTrustStore environment variable is being added to the defaultTrustStore keystore.
[INFO ] CWWKS6012I: The JSON Web Token (JWT) consumer service is available.
[INFO ] CWWKS6002I: The JSON Web Token (JWT) endpoint service is available.
[INFO ] Successfully loaded default keystore: /opt/ol/wlp/usr/servers/defaultServer/resources/security/key.p12 of type: pkcs12
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.microprofile.metrics.2.3 [176]
Unresolved requirement: Import-Package: com.ibm.wsspi.wab.configure; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.wsspi.wab.configure; bundle-symbolic-name="com.ibm.ws.app.manager.wab"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"
com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.microprofile.metrics.cdi.2.3 [177]
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
com.ibm.ws.microprofile.metrics.2.3 [176]
Unresolved requirement: Import-Package: com.ibm.wsspi.wab.configure; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.wsspi.wab.configure; bundle-symbolic-name="com.ibm.ws.app.manager.wab"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"
com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics23.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics23.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.microprofile.metrics.impl,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.microprofile.metrics.private [178]
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
com.ibm.ws.microprofile.metrics.2.3 [176]
Unresolved requirement: Import-Package: com.ibm.wsspi.wab.configure; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.wsspi.wab.configure; bundle-symbolic-name="com.ibm.ws.app.manager.wab"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"
com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.container.service.app.deploy,com.ibm.ws.container.service.state,com.ibm.ws.microprofile.metrics.impl,com.ibm.ws.ras.instrument.annotation,com.ibm.wsspi.rest.handler,javax.management,javax.servlet,javax.servlet.http,org.osgi.service.component"
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.helper; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.helper; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.microprofile.metrics,com.ibm.ws.microprofile.metrics.impl,com.ibm.ws.ras.instrument.annotation,org.eclipse.microprofile.metrics"
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.microprofile.metrics.public [179]
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.container.service.app.deploy,com.ibm.ws.container.service.state,com.ibm.ws.microprofile.metrics.impl,com.ibm.ws.ras.instrument.annotation,com.ibm.wsspi.rest.handler,javax.management,javax.servlet,javax.servlet.http,org.osgi.service.component"
com.ibm.ws.microprofile.metrics.2.3 [176]
Unresolved requirement: Import-Package: com.ibm.wsspi.wab.configure; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.wsspi.wab.configure; bundle-symbolic-name="com.ibm.ws.app.manager.wab"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"
com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.helper; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.helper; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.microprofile.metrics,com.ibm.ws.microprofile.metrics.impl,com.ibm.ws.ras.instrument.annotation,org.eclipse.microprofile.metrics"
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.microprofile.metrics.2.3.monitor [253]
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics23.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics23.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.microprofile.metrics.impl,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
com.ibm.ws.microprofile.metrics.2.3 [176]
Unresolved requirement: Import-Package: com.ibm.wsspi.wab.configure; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.wsspi.wab.configure; bundle-symbolic-name="com.ibm.ws.app.manager.wab"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"
com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.microprofile.faulttolerance.metrics.2.0 [268]
Unresolved requirement: Import-Package: com.ibm.ws.microprofile.metrics.impl; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.ws.microprofile.metrics.impl; bundle-symbolic-name="com.ibm.ws.microprofile.metrics.2.3"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"; uses:="com.ibm.websphere.ras.annotation,com.ibm.ws.ras.instrument.annotation,javax.enterprise.inject,org.eclipse.microprofile.config.spi,org.eclipse.microprofile.metrics"
com.ibm.ws.microprofile.metrics.2.3 [176]
Unresolved requirement: Import-Package: com.ibm.wsspi.wab.configure; version="[1.0.0,2.0.0)"
-> Export-Package: com.ibm.wsspi.wab.configure; bundle-symbolic-name="com.ibm.ws.app.manager.wab"; bundle-version="1.0.59.cl211220211117-1256"; version="1.0.0"
com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
[INFO ] CWWKS5500I: The MicroProfile JWT configuration [MicroProfileJwtService] was successfully processed.
[INFO ] CWWKS5500I: The MicroProfile JWT configuration [myMpJwt] was successfully processed.
[INFO ] CWWKS5500I: The MicroProfile JWT configuration [defaultMpJwt] was successfully processed.
[ERROR ] CWWKE0702E: Could not resolve module: com.ibm.ws.app.manager.wab [60]
Unresolved requirement: Import-Package: com.ibm.ws.app.manager.module.internal; version="[1.0.0,2.0.0)"
```
@jwalcorn thinks this is due to `mpReactiveMessaging-1.0` and something pulling in a MicroProfile 4.x feature. 

If you change the `Dockerfile` to read:
```Dockerfile
...
FROM openliberty/open-liberty:21.0.0.9-kernel-slim-java11-openj9-ubi
...
```
then the server starts as expected.

**Comment by jwalcorn** [comment]
I can't reproduce this issue.  I have Open Liberty 21.0.0.12, and my Trade History starts just fine.  I'm not sure what is causing the difference.  It's possible there had been a bad 21.0.0.12 build out there and IBM realized that and quietly replaced it with a working one.  Here's what I see when I start Trade History on 21.0.0.12:

jalcorn@Johns-MBP-8 trade-history % docker run -e MP_MESSAGING_CONNECTOR_LIBERTY_KAFKA_BOOTSTRAP_SERVERS=broker-4-63pby2k66qq17x9z.kafka.svc01.us-south.eventstreams.cloud.ibm.com:9093 -e MP_MESSAGING_CONNECTOR_LIBERTY_KAFKA_SASL_JAAS_CONFIG='org.apache.kafka.common.security.scram.ScramLoginModule required username=test password=test;' -e MP_MESSAGING_INCOMING_STOCKTRADER_TOPIC=stocktrader trade-history 

Launching defaultServer (Open Liberty 21.0.0.12/wlp-1.0.59.cl211220211117-1256) on Eclipse OpenJ9 VM, version 11.0.13+8 (en_US)
[AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/keystore.xml
[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/open-default-port.xml
[INFO    ] CWWKE0002I: The kernel started after 1.235 seconds
[INFO    ] CWWKF0007I: Feature update started.
[INFO    ] Aries Blueprint packages not available. So namespaces will not be registered
[INFO    ] Aries Blueprint packages not available. So namespaces will not be registered
[INFO    ] CWWKS0007I: The security service is starting...
[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
[INFO    ] DYNA1001I: WebSphere Dynamic Cache instance named baseCache initialized successfully.
[INFO    ] DYNA1071I: The cache provider default is being used.
[INFO    ] DYNA1056I: Dynamic Cache (object cache) initialized successfully.
[INFO    ] CWWKS5775I: The MicroProfile JWT version 1.1 mpConfigProxy processed successfully.
[INFO    ] CWWKS1123I: The collective authentication plugin with class name NullCollectiveAuthenticationPlugin has been activated. 
[INFO    ] CWWKS4105I: LTPA configuration is ready after 0.025 seconds.
[INFO    ] CWWKS6012I: The JSON Web Token (JWT) consumer service is available.
[INFO    ] CWWKS6002I: The JSON Web Token (JWT) endpoint service is available.
[INFO    ] Successfully loaded default keystore: /opt/ol/wlp/usr/servers/defaultServer/resources/security/key.p12 of type: pkcs12
[INFO    ] SESN8501I: The session manager did not find a persistent storage location; HttpSession objects will be stored in the local application server's memory.
[INFO    ] CWWKS5500I: The MicroProfile JWT configuration [MicroProfileJwtService] was successfully processed.
[INFO    ] CWWKS5500I: The MicroProfile JWT configuration [myMpJwt] was successfully processed.
[INFO    ] CWWKS5500I: The MicroProfile JWT configuration [defaultMpJwt] was successfully processed.
[INFO    ] CWPMI2003I: Monitoring metrics can be retrieved through mpMetrics.
[INFO    ] SRVE0169I: Loading Web Module: PublicMicroProfileMetrics.
[INFO    ] SRVE0169I: Loading Web Module: health.
[INFO    ] SRVE0169I: Loading Web Module: MicroProfileOpenAPI.
[INFO    ] SRVE0169I: Loading Web Module: OpenAPIUI.
[INFO    ] SRVE0169I: Loading Web Module: com.ibm.ws.security.jwt.
[INFO    ] SRVE0169I: Loading Web Module: ibm/api.
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/metrics/
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/openapi/ui/
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/health/
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/jwt/
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/openapi/
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/ibm/api/
[INFO    ] CWWKZ0018I: Starting application trade-history.
[INFO    ] SESN0176I: A new session context will be created for application key default_host/health
[INFO    ] SESN0176I: A new session context will be created for application key default_host/ibm/api
[INFO    ] SESN0176I: A new session context will be created for application key default_host/openapi
[INFO    ] SESN0176I: A new session context will be created for application key default_host/metrics
[INFO    ] SESN0176I: A new session context will be created for application key default_host/jwt
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] SESN0176I: A new session context will be created for application key default_host/openapi/ui
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] DYNA1056I: Dynamic Cache (object cache) initialized successfully.
[INFO    ] SRVE0242I: [com.ibm.ws.microprofile.metrics.public] [/metrics] [PublicMetricsRESTProxyServlet]: Initialization successful.
[INFO    ] SRVE0242I: [com.ibm.ws.microprofile.health.2.0] [/health] [HealthCheckReadinessServlet]: Initialization successful.
[INFO    ] SRVE0242I: [com.ibm.ws.microprofile.health.2.0] [/health] [HealthCheckServlet]: Initialization successful.
[INFO    ] SRVE0242I: [com.ibm.ws.microprofile.health.2.0] [/health] [HealthCheckLivenessServlet]: Initialization successful.
[INFO    ] SRVE0242I: [com.ibm.ws.microprofile.openapi] [/openapi] [OpenAPIServlet]: Initialization successful.
[INFO    ] SRVE9103I: A configuration file for a web server plugin was automatically generated for this server at /opt/ol/wlp/output/defaultServer/logs/state/plugin-cfg.xml.
[INFO    ] CWWKO1660I: The application trade-history was processed and an OpenAPI document was produced.
[INFO    ] WELD-000900: 3.1.8 (Final)
[INFO    ] CWWKW0750I: The following interfaces are configured as RequestScoped which will result in slower performance:  Consider using ApplicationScoped for better performance.
[INFO    ] Analyzing mediator bean: Managed Bean [class com.ibm.hybrid.cloud.sample.stocktrader.tradehistory.kafka.MPReactiveConsumer] with qualifiers [@Any @Default]
[INFO    ] Scanning Type: class com.ibm.hybrid.cloud.sample.stocktrader.tradehistory.kafka.MPReactiveConsumer
[INFO    ] Deployment done... start processing
[INFO    ] Found incoming connectors: []
[INFO    ] Found outgoing connectors: []
[INFO    ] Stream manager initializing...
[INFO    ] ConsumerConfig values: 
	auto.commit.interval.ms = 5000
	auto.offset.reset = latest
	bootstrap.servers = [broker-4-63pby2k66qq17x9z.kafka.svc01.us-south.eventstreams.cloud.ibm.com:9093]
	check.crcs = true
	client.dns.lookup = default
	client.id = trade-history-consumer
	connections.max.idle.ms = 540000
	default.api.timeout.ms = 60000
	enable.auto.commit = true
	exclude.internal.topics = true
	fetch.max.bytes = 52428800
	fetch.max.wait.ms = 500
	fetch.min.bytes = 1
	group.id = trade-history
	heartbeat.interval.ms = 3000
	interceptor.classes = []
	internal.leave.group.on.close = true
	isolation.level = read_uncommitted
	key.deserializer = class org.apache.kafka.common.serialization.StringDeserializer
	max.partition.fetch.bytes = 1048576
	max.poll.interval.ms = 300000
	max.poll.records = 500
	metadata.max.age.ms = 300000
	metric.reporters = []
	metrics.num.samples = 2
	metrics.recording.level = INFO
	metrics.sample.window.ms = 30000
	partition.assignment.strategy = [class org.apache.kafka.clients.consumer.RangeAssignor]
	receive.buffer.bytes = 65536
	reconnect.backoff.max.ms = 1000
	reconnect.backoff.ms = 50
	request.timeout.ms = 30000
	retry.backoff.ms = 100
	sasl.client.callback.handler.class = null
	sasl.jaas.config = [hidden]
	sasl.kerberos.kinit.cmd = /usr/bin/kinit
	sasl.kerberos.min.time.before.relogin = 60000
	sasl.kerberos.service.name = null
	sasl.kerberos.ticket.renew.jitter = 0.05
	sasl.kerberos.ticket.renew.window.factor = 0.8
	sasl.login.callback.handler.class = null
	sasl.login.class = null
	sasl.login.refresh.buffer.seconds = 300
	sasl.login.refresh.min.period.seconds = 60
	sasl.login.refresh.window.factor = 0.8
	sasl.login.refresh.window.jitter = 0.05
	sasl.mechanism = PLAIN
	security.protocol = SASL_SSL
	send.buffer.bytes = 131072
	session.timeout.ms = 10000
	ssl.cipher.suites = null
	ssl.enabled.protocols = [TLSv1.2]
	ssl.endpoint.identification.algorithm = HTTPS
	ssl.key.password = null
	ssl.keymanager.algorithm = SunX509
	ssl.keystore.location = null
	ssl.keystore.password = null
	ssl.keystore.type = JKS
	ssl.protocol = TLSv1.2
	ssl.provider = null
	ssl.secure.random.implementation = null
	ssl.trustmanager.algorithm = PKIX
	ssl.truststore.location = null
	ssl.truststore.password = null
	ssl.truststore.type = JKS
	value.deserializer = class org.apache.kafka.common.serialization.StringDeserializer

[INFO    ] Successfully logged in.
[INFO    ] Kafka version : 2.1.0
[INFO    ] Kafka commitId : eec43959745f444f
[INFO    ] Initializing mediators
[INFO    ] Connecting mediators
[INFO    ] Attempt to resolve com.ibm.hybrid.cloud.sample.stocktrader.tradehistory.kafka.MPReactiveConsumer#consume
[INFO    ] Connecting com.ibm.hybrid.cloud.sample.stocktrader.tradehistory.kafka.MPReactiveConsumer#consume to `stocktrader` (org.eclipse.microprofile.reactive.streams.operators.core.PublisherBuilderImpl@9dec5e7)
[INFO    ] SRVE0169I: Loading Web Module: IBM Eventstreams Kafka Sample.
[AUDIT   ] CWWKT0016I: Web application available (default_host): http://c4817a14eea4:9080/
[INFO    ] SESN0176I: A new session context will be created for application key default_host/
[INFO    ] SESN0172I: The session manager is using the Java default SecureRandom implementation for session ID generation.
[INFO    ] SRVE9103I: A configuration file for a web server plugin was automatically generated for this server at /opt/ol/wlp/output/defaultServer/logs/state/plugin-cfg.xml.
[AUDIT   ] CWWKZ0001I: Application trade-history started in 3.689 seconds.
[INFO    ] CWWKO0219I: TCP Channel defaultHttpEndpoint has been started and is now listening for requests on host *  (IPv4) port 9080.
[INFO    ] CWWKO0219I: TCP Channel defaultHttpEndpoint-ssl has been started and is now listening for requests on host *  (IPv4) port 9443.
[AUDIT   ] CWWKF0012I: The server installed the following features: [appSecurity-2.0, cdi-2.0, concurrent-1.0, distributedMap-1.0, jaxrs-2.1, jaxrsClient-2.1, jndi-1.0, json-1.0, jsonb-1.0, jsonp-1.1, jwt-1.0, microProfile-3.3, monitor-1.0, mpConfig-1.4, mpFaultTolerance-2.1, mpHealth-2.2, mpJwt-1.1, mpMetrics-2.3, mpOpenAPI-1.1, mpOpenTracing-1.3, mpReactiveMessaging-1.0, mpReactiveStreams-1.0, mpRestClient-1.4, opentracing-1.3, servlet-4.0, ssl-1.0].
[INFO    ] CWWKF0008I: Feature update completed in 6.300 seconds.
[AUDIT   ] CWWKF0011I: The defaultServer server is ready to run a smarter planet. The defaultServer server started in 7.536 seconds.

**Comment by jwalcorn** [comment]
I'm closing this and rejecting the PR.  All seems to be working fine on 21.0.0.12.  If you still see any issues, ping me on Slack or re-open the issue.
